__tests__/app.test.js

```diff
@@ -0,0 +1,50 @@
+const app = require("../app");
+const request = require("supertest");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const { articleData, commentData, topicData, userData } = require("../db/data/test-data/index");
+
+
+beforeEach(() => seed({ articleData, commentData, topicData, userData }));
+afterAll(() => db.end());
+
+
+describe("/api/topics", () => {
+  test("GET:200 sends an array of topic objects", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then((response) => {
+        expect(response.body.topics.length).toBe(3);
+        expect(response.body.topics).toEqual([
```

Comment:
You have a forEach below that is more robust than this. What if more topics were added?